cli/src/commands/branch.rs

```diff
@@ -316,6 +303,23 @@ fn cmd_branch_rename(
         ),
     )?;
 
+    let view = workspace_command.repo().view();
+    if view
+        .remote_branches_matching(
+            &StringPattern::exact(old_branch),
+            &StringPattern::everything(),
+        )
+        .any(|(_, remote_ref)| remote_ref.is_tracking())
+    {
+        writeln!(
+            ui.warning(),
+            "Warning: Branch {old_branch} is tracking remote branches which were not \
```

Comment:
s/{old_branch} is/{old_branch} has/ ?

Reply:
No longer relevant for now, https://github.com/martinvonz/jj/pull/2935#discussion_r1477439561.

(But thanks!)